roles/ncn-common-setup/tasks/gcp.yml

```diff
@@ -8,6 +8,22 @@
     owner: root
     group: root
 
+- name: backup /etc/sysconfig/network/config
+  copy:
+    src: /etc/sysconfig/network/config
+    dest: /etc/sysconfig/network/config.backup
+
+- name: use Cray DNS servers at buld-time
+  replace:
+    path: /etc/sysconfig/network/config
+    regexp: '^(NETCONFIG_DNS_STATIC_SERVERS=).*'
+    replace: '\1"172.31.84.40 172.30.84.40"'
```

Comment:
Wondering if these IPs should be variables with defaults.  Not sure that buys us anything in a practical sense, but the hard-coded IPs _here_ are making my eye twitch.